Multisig/Presentation/UI Library/QRScanner/QRCodeScannerViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  ScannerViewController.swift
+//  Multisig
+//
+//  Created by Moaaz on 4/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+protocol QRCodeScannerViewControllerDelegate {
+    func scannerViewControllerDidScan(_ code: String)
+}
+
+class QRCodeScannerViewController: UIViewController {
+    @IBOutlet weak var headerLabel: UILabel!
+    var previewLayer: AVCaptureVideoPreviewLayer!
+    var captureSession: AVCaptureSession!
+    var delegate: QRCodeScannerViewControllerDelegate?
+    var header: String?
+    
+    enum Strings {
+        static let cameraAlertTitle = NSLocalizedString("camera_title", comment: "")
+        static let cameraAlertMessage = NSLocalizedString("camera_message", comment: "")
+        static let cameraAlertCancel = NSLocalizedString("cancel", comment: "")
+        static let cameraAlertAllow = NSLocalizedString("settings", comment: "")
+        static let scannerNotSupportedTitle = NSLocalizedString("scanner_not_supported_title", comment: "")
+        static let scannerNotSupportedMessage = NSLocalizedString("scanner_not_supported_message", comment: "")
+        static let ok = NSLocalizedString("ok", comment: "")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        headerLabel.text = header
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        if (captureSession?.isRunning == false) {
```

Comment:
are parentheses here required? I think no, and parentheses-less is a preferred coding style. (Also please check other conditional statements below that have the same issue)